src/status_im/acquisition/chat.cljs

```diff
@@ -8,18 +8,20 @@
             [status-im.chat.models :as chat]))
 
 (fx/defn start-acquisition
-  [{:keys [db] :as cofx} {:keys [key] :as referrer}]
-  {:db                             (assoc-in db [:acquisition :chat-referrer key] referrer)
+  [{:keys [db]} {:keys [key id] :as referrer} public]
+  {:db                             (assoc-in db [:acquisition :chat-referrer (or key id)] referrer)
```

Comment:
I am not sure you need to save `id` in the database, it would be a public chat id, while referrer would be a public key, so it's probably not what you expect, and I don't think is used for public chats (there's no "accept" pack)

Reply:
i believe we save it in async storage , only to do not start chat on the next login